apps/insights/src/components/PriceFeeds/index.tsx

```diff
@@ -124,6 +124,27 @@ export const PriceFeeds = async () => {
           />
         </UnstyledTabPanel>
       </UnstyledTabs>
+      <div className={styles.trademarkDisclaimerSection}>
```

Comment:
We don't need a div wrapping the card, just pass the `styles.trademarkDisclaimerSection` class to the  `className` prop in the `Card` component.

While you're at it, please rename `styles.trademarkDisclaimerSection` to `styles.trademarkDisclaimerCard`